### DISCUSSION TITLE
建议增加打开即自动播放功能，谢谢

### DISCUSSION THREAD

**Discussion by bakeham** [the1812/Bilibili-Evolved#1331] [功能建议 / Ideas]
您好，
建议增加打开即自动播放功能，谢谢

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/102505320-ccdd0c00-40bc-11eb-8fe3-ff3bee7c7b35.png)

**Comment by Geeyun-JY3** [comment]
#491，#522，#542，#618，#646，#691，#1144

@the1812 感觉可以写进 #1301 了。估计那个自动连播也会不少人问，也可以加。

建议在去掉一些官方已经加上的功能后，和现有功能对官方新加入的一些功能有一定但非破坏性的影响（不知道该精准怎么描述，就如上面提到的自动连播）时，用户更新对应修复的正式版的时候给个弹窗提醒。
